docs/initial-screen.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 12
+---
+
+# Initial Screens
```

Comment:
name of the file is `initial-screen`, let's keep the same here.

And keep SDK as plural.